### ISSUE TITLE
Helm-controller may leave resources after helm-release removal

### ISSUE THREAD

**Issue by DinaBelova** [Mirantis/hmc#150]
```
{"level":"info","ts":"2024-07-24T13:57:17.294Z","msg":"uninstalled Helm release for deleted resource","controller":"helmrelease","controllerGroup":"helm.toolkit.fluxcd.io","controllerKind":"HelmRelease","HelmRelease":{"name":"ap-test","namespace":"hmc-system"},"namespace":"hmc-system","name":"ap-test","reconcileID":"e6d7f7e4-7c33-403f-98b3-a1f1b7da9aa9"}


k get secret | grep ap-test.v1
sh.helm.release.v1.ap-test.v1   
```

**Comment by slysunkin** [comment]
I think with fix for #217 this issue will be fixed as well.

**Comment by slysunkin** [comment]
It is related to #217 and should be fixed with it
